### PR TITLE
Release Notes for WebView2 119 Prerelease SDK

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article section for review:
https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes?branch=pr-en-us-2838#102106-prerelease [[gh preview](https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/relnotes-prerelease-119/microsoft-edge/webview2/release-notes.md#102106-prerelease)]

Backup rendered page in the working branch (in case PR branch doesn't trigger doc build):
https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes?branch=user%2Fmikehoffms%2Frelnotes-prerelease-119#102106-prerelease

Live article:
https://learn.microsoft.com/microsoft-edge/webview2/release-notes

Global changes to RelNotes page:
* Globally changed from:
For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version
to:
For full API compatibility, this Release version of the WebView2 SDK requires WebView2 Runtime version
* Globally changed from:
For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version
to:
For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version
* Revised the two section templates at top.
* Eliminated use of the "General" heading, and reduced the use of that heading as a parent outline level.
   * Flattened outline.  
   * Changed some "General" headings to "All platforms".
   * Changed some "General" headings to "General features".
* Consistent divider comments and blank lines above headings.
* Changed "Experimental features" headings to "Experimental APIs".
* Moved "Promotions" section to be above "Bug fixes" section; put "Bug fixes" consistently last.
   * Tightened lists in Bug Fixes sections.
* Fixed an anchor-link bug: Made a generic "Experimental APIs" section heading version-specific, b/c it's a destination for a link from [Frames article section](https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/frames?branch=pr-en-us-2838&tabs=dotnetcsharp#modifying-network-events-by-using-the-webresourcerequested-event-in-iframes).  
   * The "Experimental features" link in [live Frames article section](https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/frames?branch=pr-en-us-2838&tabs=dotnetcsharp#modifying-network-events-by-using-the-webresourcerequested-event-in-iframes) goes to wrong release.
* Removed trailing spaces.
* Removed API kind from C++ link text:
   * For readability.
   * To simplify GetFoo ("method") vs get_Foo ("property") gray area.
   * To match C++ API Ref docs, which omit "method" or "property" from heading.

AB#46571243

### CODE REVIEW

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -104,6 +104,47 @@ For full API compatibility, this version of the WebView2 SDK requires Microsoft
 <!-- end of top matter -->
 
 
+<!-- ====================================================================== -->
+## 1.0.2098-prerelease
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:108]: todo: finalize

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -105,27 +105,131 @@ For full API compatibility, this version of the WebView2 SDK requires Microsoft
...
+
+---
+
+
+<!-- ------------------------------ -->
+* Added the Frame Process Info API, enabling host applications to identify which processes are for each WebView2 control, to assess the performance impact of WebView2 on the application and on the user's device.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+<!-- todo: add for .NET, like for Win32?
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:147]: @victorhuangwq @peiche-jessica 
Should .NET tab contain these links, like Win32 has?
* `CoreWebView2Environment` Class
    * [CoreWebView2Environment.GetProcessExtendedInfos Method]()
* [CoreWebView2FrameInfo Class]()
    * [CoreWebView2FrameInfo.ParentFrameInfo Property]()
    * [CoreWebView2FrameInfo.FrameId Property]()
    * [CoreWebView2FrameInfo.FrameKind Property]()

When API Ref PR exists,
https://github.com/MicrosoftDocs/webview2-win32-reference/pulls
i can investigate

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -105,27 +105,131 @@ For full API compatibility, this version of the WebView2 SDK requires Microsoft
...
+    * [CoreWebView2FrameInfo.FrameId Property]()
+    * [CoreWebView2FrameInfo.FrameKind Property]()
+-->
+* `CoreWebView2ProcessExtendedInfo` Class
+    * [CoreWebView2ProcessExtendedInfo.AssociatedFrameInfos Property](/dotnet/api/microsoft.web.webview2.core.corewebview2processextendedinfo.associatedframeinfos?view=webview2-dotnet-1.0.2106-prerelease&preserve-view=true)
+    * [CoreWebView2ProcessExtendedInfo.ProcessInfo Property](/dotnet/api/microsoft.web.webview2.core.corewebview2processextendedinfo.processinfo?view=webview2-dotnet-1.0.2106-prerelease&preserve-view=true)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+<!-- todo: add for WinRT, like for Win32?
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:161]: @victorhuangwq @peiche-jessica 
Should WinRT tab contain these links, like Win32 has?
* `CoreWebView2Environment` Class
    * [CoreWebView2Environment.GetProcessExtendedInfos Method]()
* [CoreWebView2FrameInfo Class]()
    * [CoreWebView2FrameInfo.ParentFrameInfo Property]()
    * [CoreWebView2FrameInfo.FrameId Property]()
    * [CoreWebView2FrameInfo.FrameKind Property]()

When API Ref PR exists,
https://github.com/MicrosoftDocs/webview2-winrt-reference/pulls
i can investigate

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -105,27 +105,132 @@ For full API compatibility, this version of the WebView2 SDK requires Microsoft
...
+##### [Win32/C++](#tab/win32cpp)
+
+n/a; WinRT only
+
+---
+
+
+<!-- ------------------------------ -->
+
+* The `GetProcessExtendedInfo` API provides a snapshot collection of all actively running frames in the associated renderer process. This provides the granularity needed for the host application to know which part of WebView2 is consuming resources, such as memory or CPU usage.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:144]:
```suggestion
* The `Frame Process Info` API provides a snapshot collection of all actively running frames in the associated renderer process. This provides the granularity needed for the host application to know which part of WebView2 is consuming resources, such as memory or CPU usage.
```
@victorhuangwq @peiche-jessica - change name of broad API?
Not sure if we should call it "the GetProcessExtendedInfo API", given that the .NET tab & WinRT tab currently don't list any API type or member called GetProcessExtendedInfo(s).  Seems like either those tabs should add those API Ref links that only Win32 lists (including CoreWebView2Environment.GetProcessExtendedInfos), or, don't call it "the GetProcessExtendedInfo API".

.NET tab just lists the following; no word "get":
* CoreWebView2ProcessExtendedInfo
   * CoreWebView2ProcessExtendedInfo.AssociatedFrameInfos
   * CoreWebView2ProcessExtendedInfo.ProcessInfo

 Rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-2838&tabs=win32cpp#experimental-features

Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md]: changed to F P I

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md]: added

File: microsoft-edge/webview2/release-notes.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md]: added

**Review by peiche-jessica — APPROVED** [review]
Left comment offline.

**Review by victorhuangwq — APPROVED** [review]
LGTM!

**Pull request merged**